docs/helm-chart/index.rst

```diff
@@ -140,3 +140,22 @@ will not start as the migrations will not be run:
 This is so these CI/CD services can perform updates without issues and preserve the immutability of Kubernetes Job manifests.
 
 This also applies if you install the chart using ``--wait`` in your ``helm install`` command.
+
+.. note::
+    While deploying this Helm chart with Argo, you might encounter issues with ``DB Migrations`` not running automatically on upgrade.
+
+To ensure DB Migrations with Argo CD, you will need to:
```

Comment:
```suggestion
To ensure database migrations with Argo CD, you will need to add:
```

Would simply write "DB" out fully for clarity. Also, "Migrations" is not a name so lowercase.